step-automation-packages/step-automation-packages-support/src/main/java/step/automation/packages/deserialization/AutomationPackageParametersRegistration.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (C) 2020, exense GmbH
+ *
+ * This file is part of STEP
+ *
+ * STEP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * STEP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with STEP.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package step.automation.packages.deserialization;
+
+import step.automation.packages.model.AutomationPackageParameter;
+import step.automation.packages.model.AutomationPackageSchedule;
+
+public class AutomationPackageParametersRegistration {
```

Comment:
I see this class is also used in Junit, I assume that's why it is not directly in the controller plugin? Also how does it work for the Step runner? I'm also not sure whether this is the right location.